data/json/items/armor.json

```diff
@@ -1693,7 +1693,7 @@
     "coverage": 95,
     "encumbrance": 20,
     "material_thickness": 4,
-    "flags": [ "VARSIZE", "STURDY", "HELMET_COMPAT" ]
+    "flags": [ "VARSIZE", "STURDY", "HELMET_COMPAT", "RAINPROOF", "WATER_FRIENDLY" ]
```

Comment:
Chainmail suit, full of holes, is rainproof?

Reply:
I actually figured the rainproofing to be the leather beneath the suit, but the metal itself is also very tightly linked.